lib/command.js

```diff
@@ -226,14 +232,6 @@ module.exports = function (proto) {
       return cb(e);
     }
     proc.stdin.once('error', cb);
-    
-    proc.on('error', function(err){
```

Comment:
Moved this block for clarity. It is only effective when `bufferOutput` is `true`. See discussion in #256.